liiatools/datasets/cin_census/lds_cin_clean/validator.py

```diff
@@ -33,8 +33,8 @@ def _get_validation_error(
             )
             LAchildID_error.append(str(e))
             return event
-        if "Unexpected" in e.reason:
-            field_error.append(f"Unexpected fields found. See more information below")
+        if " expected" in e.reason:
+            field_error.append(f"Missing required fields. See more information below")
```

Comment:
Instead of dumping the whole error, you could parse out the exact missing field to make your field_error clearer.  Something like this:

```
import re
...
if " expected" in e.reason:
     regexp = re.compile(r"(?=\sexpected.*?(\w+))")
     missing_field = regexp.search(e.reason).group(1)
     field_error.append(f"Missing required field: {}.".format(missing_field))
```

Reply:
Have made these changes now, and also changed the error logs to do with LAchildID in a similar way. I've included the line number the error occurs on so LAs have enough information to make the changes